docs/user-guide/providing-feedback-and-contributing-client-side.md

```diff
@@ -78,3 +68,13 @@ Go to the [Zowe Client Kotlin SDK issue list](https://github.com/zowe/zowe-clien
 ### Zowe Client Java SDK *under development*
```

Comment:
I was given the go ahead to mark the java sdk as GA status now. Can we reflect it here as such?

Reply:
maybe remove *under development* and move the Java up the list under node.js?